narwhal/consensus/src/bullshark.rs

```diff
@@ -150,44 +150,52 @@ impl ConsensusProtocol for Bullshark {
         // Get an ordered list of past leaders that are linked to the current leader.
         debug!("Leader {:?} has enough support", leader);
         let mut committed_sub_dags = Vec::new();
+        let mut total_committed_certificates = 0;
 
         // TODO: duplicated in tusk.rs
         for leader in utils::order_leaders(&self.committee, leader, state, Self::leader)
             .iter()
             .rev()
         {
-            debug!("Previous Leader {:?} has enough support", leader);
+            let sub_dag_index = state.latest_sub_dag_index + 1;
+            let _span = error_span!("bullshark_process_sub_dag", sub_dag_index);
```

Comment:
Why an `error_span` here?

Reply:
As explained in https://github.com/MystenLabs/sui/pull/7462, having the span at `error` level ensures the span gets created and logs get annotated even if only logging at ERROR level.